packages/mobile/locales/es-419/accountScreen10.json

```diff
@@ -4,21 +4,21 @@
   "languageSettings": "Idioma",
   "localCurrencySetting": "Moneda",
   "licenses": "Licencias",
-  "removeAccountTitle": "Reiniciar Valora",
-  "removeAccountDetails": "Reiniciar borrará tu cuenta de este dispositivo. Tu cuenta mantendrá el dinero que tenga, pero solo será recuperable con tu Clave de Cuenta",
+  "removeAccountTitle": "Reinicializar Valora",
+  "removeAccountDetails": "Al reiniciar se borrará tu cuenta de este dispositivo. Tu cuenta mantendrá los fondos que tenga, pero solo la podrás recuperar con tu Clave de Cuenta.",
   "legal": "Legal",
   "securityAndData": "Seguridad y datos",
-  "requirePinOnAppOpen": "Requerir PIN en la aplicación abierta",
+  "requirePinOnAppOpen": "Requerir PIN cuando al app esté abierta",
   "analytics": "Estadisticas de uso",
-  "shareAnalytics": "Compartir estadisticas de uso",
-  "shareAnalytics_detail": "Recopilamos datos anónimos sobre cómo utiliza {{appName}} para ayudar a mejorar la aplicación para todos.",
+  "shareAnalytics": "Compartir estadísticas de uso",
+  "shareAnalytics_detail": "Recopilamos datos anónimos sobre cómo utilizas {{appName}} para mejorar la aplicación para todos.",
   "dataSaver": "Ahorro de Datos",
   "enableDataSaver": "Habilitar Ahorro de Datos",
   "dataSaverDetail": "El modo de Ahorro de Datos te permite comunicarte con la Red Celo a través de un nodo confiable. Puedes cambiar este modo en la configuración de la aplicación.",
   "restartModalSwitchOff": {
-    "header": "Reiniciar para apagar Ahorro de Datos",
-    "body": "Para encender y apagar Ahorro de Datos repetidamente, deberá reiniciar la aplicación y volver a esta pantalla.",
-    "restart": "Reiniciar para alternar"
+    "header": "Reinicializar para apagar Ahorro de Datos",
+    "body": "Para encender y apagar Ahorro de Datos repetidamente, deberá reinicializar la aplicación y volver a esta pantalla.",
+    "restart": "Reinicializar para alternar"
```

Comment:
Hmm is `Reinicializar` the right word here? From what I understand this is referring to the need of *restarting* the app to toggle the data saver mode, which is different from *re-initializing* the app (i.e. re-entering the backup key). 

The previous word `Reiniciar` seems more appropriate to me